TGC Week_2 Google Page

SJ-SEO opened the pull request:
- html 코드 적용
- css 코드 적용
  - 브라우저 크기 반응성을 고려 한 float 및 text-align 사용 배치
- js 동작 설정
  - Google 이미지 클릭 시 현재 페이지 새로고침
  - gmail, images 글씨 클릭 시 적절한 페이지로 redirect
  - search 기능

SJ-SEO committed:
Test commit

SJ-SEO committed:
feat : Google Wnd 제작

- html 코드 적용
- css 코드 적용
- js 동작 설정. search, redirect, alert

creativeYS commented on practice01/google.html:
```diff
@@ -5,5 +5,26 @@
+		</div>
+		<div class="search_bar">
+			<input id="google_search_button" class="search_button" type="submit" value="Google Search" onclick={onSearch();}>
+			<input class="search_button" type="submit" value="I'm Feeling Lucky" onclick={onLucky();}>
```
onclick를 html 코드에 직접 부여하는 것보다
js코드에서 window.onLoad 할때 붙여보는건 어떨까요?
코드호출에 대한 규칙 및 적용까지 js 코드에 위임하고 html은 정말 뼈대만 만드는 것입니다.

`
var el = document.getElementById("id");
el.onclick = onSearch
`

creativeYS commented on practice01/google.html:
```diff
@@ -5,5 +5,26 @@
+				<a onclick={onClickedGmail();}><p>Gmail</p></a>
+			</div>
+			<div class="nav_text_box">
+				<a onclick={onClickedImages();}><p>Images</p></a>
```
a 태그를 사용한다면 문서를 연결하는 표준 방법으로 사용하는게 좋습니다.
그래야 다양한 표준적인 상황에 대응이 가능하거든요~
`<a href="https://naver.com>link</a>`

이렇게하면 해당 링크에 마우스를 가져다대면 손가락으로 바뀝니다.
반면에 onlick 이벤트로 javascript 코드를 삽입한 경우는 마치 텍스트처럼, 인식해서 손가락이 안나오죠
사용자가 이것이 링크라고 판단하기 어렵게 만드는 부분이 됩니다.

creativeYS commented on practice01/google.html:
```diff
@@ -5,5 +5,26 @@
+	<div id="body">
+		<img id="logo" src="google.png" onclick={onClickedGoogleImg();}>
+		<div class="search_box">
+			<input id="text_box" type="text" >
```
재미삼아서 검색창에서 엔터를 쳤을 때 검색결과로 리다이렉트 되도록하는 것도 해보면 어떨까요?

SJ-SEO replied:
@creativeYS  onkeypress는 vscode에서는 쓰는걸 권장하지 않더라구요
onkeyup로 일단 넣긴 했는데 더 좋은 방법이 있을까요

creativeYS replied:
ㅎㅎㅎ press는 누르는 동안이라 그런거 같네요
생각하신 대로 onkeydown, onkeyup 하나를 필요에 따라 쓰면 될것 같습니다 :)

SJ-SEO committed:
impv : 코드리뷰 피드백 반영

- a 태그 redirect url 직접 부여
- html 코드 onclick 이벤트 js onload 로 이동
- image url a 태그로 이동

creativeYS commented on practice01/google.html:
```diff
@@ -8,22 +8,24 @@
 		<div class="search_bar">
 			<input id="google_search_button" class="search_button" type="submit" value="Google Search" onclick={onSearch();}>
-			<input class="search_button" type="submit" value="I'm Feeling Lucky" onclick={onLucky();}>
+			<input id="lucky_button" class="search_button" type="submit" value="I'm Feeling Lucky" onclick={onLucky();}>
```
아래 js 파일에서 onclick 에 대한 이벤트 처리 적용까지 담당하였으므로 이제 html 에 onclick=... 는 제거해도 되겠네요~ ㅎㅎ

creativeYS commented:
이쁘게 잘 나오네요~ ㅎㅎㅎ 이제 병합하셔도 됩니다.
추가로 더 필요하신 부분은 관심있으면 google.com 참고하여 더 해보는 것은 선택입니다! ㅎㅎㅎ

SJ-SEO committed:
impv : 불필요 js 함수 삭제

- 불필요 js 함수 삭제

SJ-SEO committed:
impv : enter key press 방식 변경

- 다른 키 눌러도 진행되는 onKeyUp 사용하지 않고 Enter만 적용
